superset/templates/superset/basic.html

```diff
@@ -63,16 +76,66 @@
 
     {% block body %}
       <div id="app" data-bootstrap="{{ bootstrap_data }}">
-        <img
-          src="{{ assets_prefix }}/static/assets/images/loading.gif"
-          style="
-            width: 50px;
-            position: absolute;
-            top: 50%;
-            left: 50%;
-            transform: translate(-50%, -50%);
-          "
-        />
+      <div class="spinner">
```

Comment:
Oops... I thought I already pulled in the SVG code here from the actual SVG file, rather than duplicating the code.

Reply:
This could basically be replaced by
`{{ include('path/to/spinner.svg') }}` I think.